modal/exception.py

```diff
@@ -213,3 +216,9 @@ class ModuleNotMountable(Exception):
 
 class ClientClosed(Error):
     pass
+
+
+def print_server_warnings(server_warnings: Iterable[api_pb2.Warning]):
+    ALARM_EMOJI = chr(0x1F6A8)
+    for warning in server_warnings:
+        warnings.warn_explicit(f"{ALARM_EMOJI} {warning} {ALARM_EMOJI}", DeprecationError, "<unknown>", 0)
```

Comment:
A couple comments:

- Should we filter the warnings here to those with a client deprecation type?
- Since we now show the warnings with a nice highlighted rich box we might not need the alarm emoji anymore
- The `modal.exception` module is functionally public (users may need to catch specific exception types) so we should IMO avoid adding other functions not intended to be user-facing to it. I added a new `modal._utils.deprecation` module in my parameter renaming PR so I can just move this function there after you merge.

Reply:
yeah all good suggestions, i can accommodate!